### PR TITLE
Added spike strip preset

### PR DESCRIPTION
**Pull request by animesh-007**
added spike strip preset fixes issue #7347

### CODE REVIEW

File: data/presets/presets/barrier/spike.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "icon": "maki-fence",
+    
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "barrier": "spike"
+    },
+    "name": "Spike",
```
Comment (1ec5) [inline data/presets/presets/barrier/spike.json]: These things go by many names, but I’m unaware of it being called simply “spike”. The most common term seems to be “spike strip”.

Comment (animesh-007) [reply data/presets/presets/barrier/spike.json]: Thank you, I just renamed it to "spike strip". Any other suggestions.

Comment (bhousel) [reply data/presets/presets/barrier/spike.json]: ok, so here is the documentation:
https://wiki.openstreetmap.org/wiki/Tag:barrier=spikes

So: 
The tags should be `"barrier": "spikes"`.  
The filename should be `spikes.json`.
The name should be `"Spike Strip"`.  
The geometry should be `["vertex"]`.

Comment (animesh-007) [reply data/presets/presets/barrier/spike.json]: Made these changes in the file.

Comment (bhousel) [reply data/presets/presets/barrier/spike.json]: > Made those changes in the file.

Sorry but the file is still not named correctly or using the correct tag.  I'm going to close this PR.

**Pull request closed without merging**